<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleDescription">
   <string>JJ/MM/AAAA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="dateEdit_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>761</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000;
  font-size: 12px;</string>
      </property>
      <property name="text">
       <string>Recerche par CIN_employe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAjouter">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>490</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>320</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000;
  font-size: 16px;</string>
      </property>
      <property name="text">
       <string>Debut Conge:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>270</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000;
  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Id_Conge:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeIdRT">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;


}


QLineEdit::hover
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}


QLineEdit::focus
{
	
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../1tap.png</normaloff>../../../1tap.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>160</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000;
  font-size: 12px;</string>
      </property>
      <property name="text">
       <string>Recerche par Id_conge</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeTitreRT">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;


}


QLineEdit::hover
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}


QLineEdit::focus
{
	
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>170</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Tri PAR DATE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Tri PAR ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LeId">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>270</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;


}


QLineEdit::hover
{
	background-color: #2c3e50;
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}


QLineEdit::focus
{
	
	color: #fcfcfc;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;


}</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>320</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(44, 62, 80, 255),stop:1 rgba(3, 2, 5, 255));
    padding-left: 6px;
	border: 1px solid #1d1d1d;
    color: #fff;
    height: 20px;

}</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>631</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
	font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    background-color: #2c3e50;
	color: #ffffff;
    padding: 4px;
    font-size: 10pt;
    border-style: none;
    border-bottom: 1px solid #fffff8;
    border-right: 1px solid #fffff8;

}

QHeaderView::section:horizontal
{
    border-top: 1px solid #fffff8;
}

QHeaderView::section:vertical
{
    border-left: 1px solid #fffff8;
}
</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>370</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(44, 65, 80, 255),stop:1 rgba(3, 2, 5, 255));
    padding-left: 6px;
	border: 1px solid #1d1d1d;
    color: #fff;
    height: 20px;

}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>370</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000;
  font-size: 16px;</string>
      </property>
      <property name="text">
       <string>Fin Conge:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonModifier">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>500</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSupprimer">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>500</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>410</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000;
  font-size: 16px;</string>
      </property>
      <property name="text">
       <string>CIN_Emp</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>420</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(44,62, 80, 255),stop:1 rgba(3, 2, 5, 255));
    padding-left: 6px;
	border: 1px solid #1d1d1d;
    color: #fff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:on
{
    background-color: #4a5157;
	color: #fff;

}


QComboBox QAbstractItemView
{
    background-color: #e8d9d7;
    color: #000;
    selection-background-color: #aa7e82;
	selection-color: #fff;
    outline: 0;

}


QComboBox::drop-down
{
	background-color: #4a5157;
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

}


QComboBox::down-arrow
{
    image: url(://arrow-down.png);
    width: 8px;
    height: 8px;

}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="conge">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>conge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="employee">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Statistiques_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>20</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1251</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/img/policeman-background_48369-15139.jpg) 1 1 1 1 stretch stretch;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="styleEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>styleEdit</zorder>
     <zorder>logo</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButtonAjouter</zorder>
     <zorder>label_11</zorder>
     <zorder>label_2</zorder>
     <zorder>LeIdRT</zorder>
     <zorder>label_9</zorder>
     <zorder>LeTitreRT</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton</zorder>
     <zorder>LeId</zorder>
     <zorder>dateEdit</zorder>
     <zorder>tableView</zorder>
     <zorder>dateEdit_2</zorder>
     <zorder>label_12</zorder>
     <zorder>pushButtonModifier</zorder>
     <zorder>pushButtonSupprimer</zorder>
     <zorder>label</zorder>
     <zorder>comboBox</zorder>
     <zorder>conge</zorder>
     <zorder>employee</zorder>
     <zorder>Statistiques_3</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="Statistiques_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calendrier_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="styleEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="chartsWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>1261</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="barchartButton">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Barchart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="piechartButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-family: &quot;Roboto&quot;, sans-serif;
    text-transform: uppercase;
    outline: 0;
    background: #2c3e50;
    color: #FFFFFF;
    width: 100%;
    border: 0;
    padding: 15px;
    font-size: 14px;
    -webkit-transition: all 0.3 ease;
    transition: all 0.3 ease;
    cursor: pointer;

}
QPushButton:hover
{

    background: #3498db;
}




</string>
      </property>
      <property name="text">
       <string>Piechart</string>
      </property>
     </widget>
     <zorder>styleEdit_2</zorder>
     <zorder>calendrier_2</zorder>
     <zorder>Statistiques_2</zorder>
     <zorder>chartsWidget</zorder>
     <zorder>barchartButton</zorder>
     <zorder>piechartButton</zorder>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
